addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -54,6 +57,11 @@ export class Attachment extends Record {
         return this.mimetype && this.mimetype.startsWith("application/pdf");
     }
 
+    get dateString() {
+        const datetime = deserializeDateTime(this.create_date);
+        return `${datetime.monthLong}, ${datetime.year}`;
+    }
+
```

Comment:
Name `dateString` is a bit obscure in the actual formatting of string. What about `monthYear`? "String" doesn't had anything, let the typing tell it's a string, rather than put it in name of getter